Code font adaptation

Dscano opened the pull request:
This PR fixes the [observation](https://github.com/p4lang/p4-spec/issues/1298#issuecomment-2269971939).

Dscano committed:
Solved: Code block font adaptation

Dscano committed:
Merge branch 'main' of github.com:Dscano/learn-asciidoc into main

Dscano committed:
Merge branch 'jafingerhut:main' into main

jafingerhut reviewed (approved):
Looks reasonable to me, whatever it does :-)

Dscano commented:
> Looks reasonable to me, whatever it does :-)

Hi Any,
I have addressed the comment from Chris. I changed the font size and padding for the code block.
